simulator/cctree.cpp

```diff
@@ -1,11 +1,11 @@
-#include "cctree.h"
-#include "method.h"
-int debug = 0;
-
-
-void CCTree::handle_object_allocation(int object_id, int size, std::string type,
-                                      int thread_id, int method_id) {
+#include "simulator/cctree.h"
+#include "simulator/method.h"
 
+namespace et_simulator {
+int debug = 0;
```

Comment:
It seems like this `debug` constant is being used in lieu of log levels (e.g. INFO, ERROR, WARNING). This should be replaced but I get if you'd rather do this in a future pull request.

Reply:
Yeah, this is to preserve the code. I have another branch for glog :-)